packages/slider/src/mwc-slider-base.ts

```diff
@@ -174,10 +170,21 @@ export class SliderBase extends FormElement {
           this.thumbContainer.style.setProperty(propertyName, value),
       setTrackStyleProperty: (propertyName: string, value: string) =>
           this.trackElement.style.setProperty(propertyName, value),
-      setMarkerValue: (value: number) => this.pinMarker.innerText =
-          value.toString(),
-      setTrackMarkers: (_step: number, _max: number, _min: number) => {
-        // TODO(aomarks) What are we supposed to do here?
+      setMarkerValue: (value: number) => this.pinMarkerText = value.toString(),
+      setTrackMarkers: (step, max, min) => {
+        const stepStr = step.toLocaleString();
+        const maxStr = max.toLocaleString();
+        const minStr = min.toLocaleString();
+        // keep calculation in css for better rounding/subpixel behavior
+        const markerAmount = `((${maxStr} - ${minStr}) / ${stepStr})`;
+        const markerWidth = '2px';
+        const markerBkgdImage = `linear-gradient(to right, currentColor ${
+            markerWidth}, transparent 0)`;
+        const markerBkgdLayout = `0 center / calc((100% - ${markerWidth}) / ${
```

Comment:
This is starting to get a little complicated. Can you put a comment in here about what this calculation is doing?

Reply:
Sure, I hate it, but it's exactly what they do in the component. It basically draws the notches on the slider if you enable notches and steps. It's more performant, but ugly

Reply:
done